"""
Generating a ValueError
"""

# ValueError
my_tuple = [1, 2, 3, 4, 5]
my_tuple.remove([1])
print(my_tuple[2] + "x")




#
my_list = list(my_tuple)






